src/main/java/quotify_app/app/AppBuilder.java

```diff
@@ -42,6 +44,7 @@ public AppBuilder() {
         comparatorFactory.setUpController(viewManagerModel);
         currentPriceFactory.setUpController(viewManagerModel);
         futurePriceFactory.setUpController(viewManagerModel);
+        landingFactory.setUpController(viewManagerModel);
```

Comment:
this is where you'd be passing in whatever necessary arguments you need for setting up the controller: i.e. the areadataaccessobject, propertydataaccessobject, areacache, along iwth the viewmanagermodel

Just instantiate the above 3 with the other class variables

Reply:
done